CONTRIBUTORS

```diff
@@ -205,6 +216,7 @@ tomscytale
 Trikke76
 truki
 tYYGH
+Vanav
 veger
 vfoucault
 vmalloc
```

Comment:
should we maybe use something like [this](https://github.com/marketplace/actions/auto-add-contributors) to auto-maintain this?

Reply:
i also add issue reporters and folks who take part in discussions in comments. Does the tool support it? Also I see security tokens mentioned, would it be safe to expose it?
Also I like doing it periodically. It reminds contributors that we're happy to see their contributions after some time passed.

Sorry for the noise folks, I should've thanked you after this PR gets merged

Reply:
ah, no it doesn't cover it all. i withdraw the suggestion. carry on :)